Makefile.pre.in

```diff
@@ -300,32 +300,24 @@ LIBFFI_INCLUDEDIR=	@LIBFFI_INCLUDEDIR@
 # Parser
 
 PEGEN_OBJS=		\
-		Parser/pegen/pegen.o \
-		Parser/pegen/parse.o \
-		Parser/pegen/parse_string.o \
-		Parser/pegen/peg_api.o
+		Parser/pegen.o \
+		Parser/parse.o \
+		Parser/parse_string.o \
```

Comment:
Can we,rename this? Themcommon start with pegen.c always,slows down Emacs filename completion.

Reply:
What about `string_parser.py`?

Reply:
Do you want to rename `parse.c` as well?

Reply:
What about `pegen.c`, `parser`, `string_parser`?

Reply:
Yes please.

Reply:
Done in e11e688